Userland/Libraries/LibTLS/HandshakeClient.cpp

```diff
@@ -224,6 +226,43 @@ void TLSv12::build_rsa_pre_master_secret(PacketBuilder& builder)
     builder.append(outbuf);
 }
 
+void TLSv12::build_dhe_rsa_pre_master_secret(PacketBuilder& builder)
+{
+    auto dh = m_context.server_diffie_hellman_params;
```

Comment:
Let's avoid this copy, I don't think you meant to copy it anyway?

Reply:
Totally meant to take a reference here. Good catch.